yarn.lock

```diff
@@ -0,0 +1,10596 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
appear this addon was using yarn under the hood but presence of package-lock.json was misleading